app/src/main/java/io/gnosis/safe/ui/settings/view/NamedAddressItem.kt

```diff
@@ -52,13 +52,13 @@ class NamedAddressItem @JvmOverloads constructor(
             field = value
         }
 
-    var name: String? = null
+    var name: Int? = null
         set(value) {
-            if (value.isNullOrBlank()) {
+            if (value == null) {
```

Comment:
wouldn't the elvis operator be nicer here?
```
value?.let{
  binding.name.visible(true)
  binding.name.setText(it)
} ?: run {
  binding.name.visible(false)
}
```

Reply:
Is `run` necessary?